internal/oci/runtime_vm.go

```diff
@@ -155,7 +152,7 @@ func (r *runtimeVM) CreateContainer(c *Container, cgroupParent string) (retErr e
 	createdCh := make(chan error)
 	go func() {
 		// Create the container
-		if resp, err := r.task.Create(r.ctx, request); err != nil {
+		if resp, err := r.task.Create(namespaces.WithNamespace(context.Background(), namespaces.Default), request); err != nil {
```

Comment:
why this change? also, why aren't we passing the ctx to CreateContainer as well?

Reply:
yes, missed to replace it with ctx, fixed now. 
ctx is updated with Namespace in startRuntimeDaemon, so its not needed anymore in other places. is my understanding right ?